geniza/corpus/metadata_export.py

```diff
@@ -74,6 +78,7 @@ def get_queryset(self):
                 "secondary_languages",
                 "log_entries",
                 "log_entries__user",
+                "dating_set",
```

Comment:
Nice. Were you able to confirm the export is able to prefetch all the related data and doesn't have to make any new queries per item?

Reply:
Confirmed! 👍